Assignment_3/PingHost.java

```diff
@@ -0,0 +1,70 @@
+package com.assignments.java;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class PingHost {
+
+    /**
+     * It takes host address as input and
+     * prints out the median of time taken to ping the given host address
+     *
+     * @param args
+     * @throws IOException
+     */
+    public static void main(String[] args) throws IOException {
+        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
+            System.out.println("Enter a host address:");
+            String host = bufferedReader.readLine();
+            System.out.println(findMedian(host));
+
+    }
+
+    /**
+     * It prints the median of time taken to ping the given host using the system utility command 'ping'.
+     * It finds the median from the given list of time taken
+     * @param host
+     * @return median
+     */
+    public static Float findMedian(String host) {
```

Comment:
have a proper function name. also best practise of a function is it should serve one functionality. Function should not be lengthy. If you feel a function is lengthy break into multiple functions